API.md

```diff
@@ -584,6 +587,43 @@ Controls whether the ExoPlayer shutter view (black screen while loading) is enab
 
 Platforms: Android
 
+#### shutterColor
+Apply color to shutter view, if you see black flashes before video start then set shutterColor='transparent'
+
+won't work if useTextureView={false}
+
+* **black (default)**
+
+Platforms: Android
+
+#### frameQuality
+Use this prop to set quality of Base64 encoded string image obtained by onFrameChange callback. 
```

Comment:
can you info that onFrameChange will be reported with same frequency than onProgress ?
No sure this is good, time position should be fast, frame callback can be slower

Reply:
Tested on lower end devices the frame drop will increase as frameQuality increase. But keeping frameQuality=30 will feasible and one can get all the frames.

This functionality is needed in Flatlist as user can start from where he left.
frameCallback will be slower but the last frame will be the same where user paused the video that's why I have used runnable thread.